Woof/Woof/View/SitterFlow/ScheduleView/ScheduleView.swift

```diff
@@ -4,11 +4,15 @@ import SwiftUI
 struct ScheduleView: View {
     var body: some View {
         NavigationView {
-            Text("My schedule")
-                .navigationTitle("My Schedule")
+            Text(navigationTitle)
+                .navigationTitle(navigationTitle)
```

Comment:
Do we use the same value for both cases?

Reply:
Fixed with 2afb5c0